ios/brave-ios/App/l10n/tools/download-translations-from-transifex.swift

```diff
@@ -87,62 +129,86 @@ guard let apiKey = ProcessInfo.processInfo.environment["PASSWORD"] else {
   }
 }
 
-@Sendable func cleanupAndWriteResourceToDisk(xliffData: Data, languageCode: String) async {
+@Sendable func cleanupAndWriteResourceToDisk(xliffData: Data, languageCodes: [String]) async {
   let fileManager = FileManager.default
-  let xliffURL = fileManager.temporaryDirectory.appendingPathComponent("\(languageCode).xliff")
-  do {
-    try xliffData.write(to: xliffURL)
-  } catch {
-    print("ERROR: Failed to write xliff file")
-    exit(EXIT_FAILURE)
-  }
+  for languageCode in languageCodes {
+    let xliffURL = fileManager.temporaryDirectory.appendingPathComponent("\(languageCode).xliff")
+    do {
+      try xliffData.write(to: xliffURL)
 
-  let process = Process()
-  process.executableURL = URL(fileURLWithPath: "/usr/bin/env")
-  process.arguments = ["python3", "xliff-cleanup.py", xliffURL.path]
-  do {
-    try process.run()
-    process.waitUntilExit()
-  } catch {
-    print("Failed to run python process to cleanup xliff")
+      // If additional targets exist we have to update target language in the xliff xml.
+      if languageCodes.count > 1 {
+        try updateTargetLanguage(inXLIFFFileAtPath: xliffURL.path, newLanguage: languageCode)
+      }
```

Comment:
Seems like you may as well alter the data directly instead of writing the xliff file to disk then reading from it and rewriting again. e.g.

```swift
if langaugeCodes > 1 {
  let alteredXLIFFData = try updatingTargetLanguage(xliffData, newLanguage: languageCode)
  try alteredXLIFFData.write(to: xliffURL)
} else {
  try xliffData.write(to: xliffURL)
}
```